evm/src/lib.rs

```diff
@@ -41,6 +41,11 @@ pub static CALLER: Address = H160([
     0x30, 0x9D, 0x1F, 0x38,
 ]);
 
+/// Stores the default test contract address: 0xb4c79daB8f259C7Aee6E5b2Aa729821864227e84
+pub static TEST_CONTRACT_ADDRESS: Address = H160([
```

Comment:
i think also possible

```suggestion
pub const TEST_CONTRACT_ADDRESS: Address = H160([
```